classes/04class/exercises/c04-iac01/zarajoy/_provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+  version = "~> 2.0"
+  region  = "ap-southeast-2"
```

Comment:
didn't want to use a variable for the region ?

Reply:
fixed it. i was having issues with that variable so i took it out, but it was due to me spelling 'region' different every time i typed it (reigon, region, reigion -.-)